deploy/backplane/lpsre/rhoam/01-rhoam-lpsre-admins-project.ClusterRole.yaml

```diff
@@ -129,4 +129,31 @@ rules:
   resources:
   - deployments/scale
   verbs:
-  - patch
\ No newline at end of file
+  - patch
+# LP SRE can restart deployments/statefulsets/deploymentconfigs
+- apiGroups:
+  - apps
+  - extensions
+  resources:
+  - deployments
+  verbs:
+  - patch
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
```

Comment:
Does RHOAM use stateful sets? Just want to explicitly confirm as the card doesn't mention it.

Reply:
See comment for deploymentconfigs.